.babelrc

```diff
@@ -1,18 +1,25 @@
 {
   "plugins": [
-    "styled-components"
+    "styled-components",
+    "transform-object-rest-spread",
+    "transform-class-properties",
+    "transform-runtime"
   ],
   "presets": [
     [
       "env",
       {
-        "modules": false
+        "targets": {
+          "browsers": ["last 2 versions"]
+        }
       }
     ],
-    "react",
-    "stage-0"
+    "react"
   ],
   "env": {
+    "development": {
+      "presets": ["stage-0"]
```

Comment:
You sure about this? It should not have changed, should be used cross-env.

Reply:
This was a strange one...try removing it and running `yarn start` and `React Intl` starts complaining. I didn't have time to look into it...but agree it's strange.